lib/spack/spack/compiler.py

```diff
@@ -294,6 +294,8 @@ def determine_implicit_rpaths(cls, paths):
                                       output=str, error=str))  # str for py2
 
             return cls.parse_implicit_rpaths(output)
+        except spack.util.executable.ProcessError:
+            pass
```

Comment:
Make this issue a warning or a debug statement so that we can at least see that it happened.  I'm not completely sure *why* #12732 fails the way it does.